modules/services/nextcloud-client.nix

```diff
@@ -17,6 +17,13 @@ in {
         defaultText = literalExample "pkgs.nextcloud-client";
         description = "The package to use for the nextcloud client binary.";
       };
+
+      runInBackground = mkOption {
+        type = types.bool;
+        default = false;
+        defaultText = literalExample "false";
```

Comment:
I don't think you need `defaultText` when the default is a simple boolean.

Reply:
Yeah, you're right that is a bit useless.